tests/unit/lms/models/application_instance_test.py

```diff
@@ -35,23 +35,22 @@ def test_provisioning_is_not_nullable(self, db_session, application_instance):
 
         assert application_instance.provisioning is not None
 
-    def test_settings_defaults_to_an_empty_dict(self, application_instance):
+    def test_settings_defaults_to_None(self, application_instance):
         settings = application_instance.settings
 
-        assert isinstance(settings, ApplicationSettings)
-        assert settings.data == {}
+        assert settings is None
 
     def test_settings_can_be_retrieved(self, application_instance):
-        application_instance._settings = {"group": {"key": "value"}}
+        application_instance.settings = {"group": {"key": "value"}}
```

Comment:
This actually sets `settings` to an `ApplicationSettings` object, not a dict, which is pretty handy